### DISCUSSION TITLE
AttentionMe 프로젝트 Workflow 토의입니다.

### DISCUSSION THREAD

**Discussion by wwwwje2008** [Looking-4-Attention/AttentionMe#1] [General]
AttentionMe: 이미지 필터 라이브러리

main에서 semantic segmentation 부분 개발하고 거기에서 기능 추가하는 식으로 진행하면 좋을 것 같아요!

댓글로 추가할만한 기능 아이디어 있으면 남겨주세요

**Comment by wwwwje2008** [comment]
저는 객체 확대하는 기능 있으면 좋을 것 같아용

**Reply by iamdami** [reply]
오 객체 확대 아이디어 좋네요!

**Reply by Namhoon-Cho** [reply]
좋습니다. 이는 매우 창의적이라 할 수는 없으나, 사람을 부각시키기에 탁월한 기능입니다!

추신. segmentation을 활용한 객체 확대 기능이라면, 사람을 단순 확대하는 것인가요, 아니면 선택해 확대할 수 있나요?

**Reply by hyun1227** [reply]
저도 좋아요

**Comment by Namhoon-Cho** [comment]
사람을 강조하려면 만화적 허용 또한 안전한, 좋은 방법이라 생각합니다. 좌표를 계산해서 화살표를 두는 것은 어떨까요? 화살표가 옆에서 가리키고 있는 사람에게 관심이 쏠릴 것으로 예상합니다.

**Reply by hyun1227** [reply]
스티커 붙이는 거 같고 강조도 잘 될 것 같아서 좋은 것 같습니다

**Comment by hyun1227** [comment]
객체에 대해서 효과를 주는 것도 좋지만 배경의 밝기를 조절해서 효과를 줘도 좋을 것 같아요 
오히려 배경의 밝기로 이내 강조의 효과도 줄 수 있다고 생각합니다

**Reply by Namhoon-Cho** [reply]
밝은 것, 어두운 것 두 가지 버전이 있을 수 있겠네요!

**Reply by wwwwje2008** [reply]
밝게 하는거랑 어둡게 하는건 똑같은 알고리즘으로 구현 가능할 것 같은데 함수 하나로 만들면 좋을 것 같아용

**Comment by iamdami** [comment]
세그멘테이션한 결과 지정한 객체를 crop해서 저장하는 기능도 있으면 좋을 것 같아요!

**Reply by Namhoon-Cho** [reply]
훌륭합니다!

**Comment by Namhoon-Cho** [comment]
그럼 지금 나온 기능이, 세그멘테이션 한 결과 지정한 객체를
1. 확대한다
2. 배경을 밝게/어둡게 한다
3. 화살표로 가리킨다
4. crop 기능
이렇게 나뉘겠네요! 각자 한 부분을 개발해보도록 합시다~

**Reply by Namhoon-Cho** [reply]
참고로 제 역할을 다한 디스커션은 close할 수 있습니다.
더 이상 얘기할 게 없으면 issue처럼 close가 가능하단 점 참고 바랍니다~
